src/SwiftPackage.ts

```diff
@@ -215,9 +216,15 @@ export class SwiftPackage implements PackageContents {
         toolchain: SwiftToolchain
     ): Promise<SwiftPackageState> {
         try {
-            let { stdout } = await execSwift(["package", "describe", "--type", "json"], toolchain, {
-                cwd: folder.fsPath,
-            });
+            let { stdout } = await execSwift(
+                ["package"]
+                    .concat(configuration.buildArguments)
+                    .concat(["describe", "--type", "json"]),
```

Comment:
I prefer the `["package", ...configuration.buildArguments, "describe", "--type", "json"] format, but I guess `concat` works

Reply:
Same, we should be consistent about using the spread operator

Reply:
ACK, ya no problem spreading